engine/swarm/stack-deploy.md

```diff
@@ -116,9 +116,7 @@ counter whenever you visit it.
 5.  Create a file called `docker-compose.yml` and paste this in:
 
     ```none
-    version: "{{ site.compose_file_v3 }}"
-
-    services:
+        services:
```

Comment:
Indentation seems off for this key - I think it should hug left. But I noticed this page isn't even in the TOC... I'll check why...

```suggestion
    services:
```